filter.imp

```diff
@@ -1,13 +1,119 @@
 [
+   # If an architecture does not provide its own copy of asm/foo.h, then it will get an
+   # asm-generic/foo.h which can still be included as asm/foo,h
```

Comment:
Fix the `,` near the end; should be a `.`.

Use proper punctuation in comments (end them with periods) for this comment and the rest below.